stdweb-internal-runtime/src/runtime.js

```diff
@@ -212,18 +207,43 @@ Module.STDWEB_PRIVATE.serialize_array = function serialize_array( address, value
     }
 };
 
-Module.STDWEB_PRIVATE.from_js = function from_js( address, value ) {
-    var kind = Object.prototype.toString.call( value );
-    if( kind === "[object String]" ) {
+if ( typeof TextEncoder === "function" ) {
```

Comment:
This is just personal preference, but I find `typeof` operator a bit verbose. Using the `in` operator to check the window object feels cleaner
```suggestion
if ( 'TextEncoder' in window ) {
```

Reply:
Under Node.js you don't have a `window` object and we want to stay compatible with it, so in this case the `typeof` version is better. (:

Although unfortunately under Node.js you need to import `TextEncoder` from `util` to use it so it will still use the old encoder/decoder, which reminds me - @Pauan I'm fine with merging this PR as-is, however would you be interesting in adding Node.js support? (Which should be relatively simple all things considered.)

Reply:
ah, didn't realize y'all were targeting both browser and Node.js

Reply:
@koute Since it wasn't much code, I went ahead and added in support for older Nodes (before v11).

Reply:
Oh, I had no idea; that's good to know!